fahrschein-example/src/main/java/org/zalando/fahrschein/example/Main.java

```diff
@@ -35,227 +35,264 @@
 import org.zalando.fahrschein.http.simple.SimpleRequestFactory;
 import org.zalando.fahrschein.http.spring.SpringRequestFactory;
 import org.zalando.fahrschein.inmemory.InMemoryCursorManager;
+import org.zalando.fahrschein.opentelemetry.OpenTelemetryWrapper;
 import org.zalando.jackson.datatype.money.MoneyModule;
 
-import java.io.IOException;
-import java.net.URI;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-import java.util.concurrent.TimeUnit;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
 
-import static java.util.Arrays.asList;
-import static org.zalando.fahrschein.AuthorizationBuilder.authorization;
-import static org.zalando.fahrschein.domain.Authorization.AuthorizationAttribute.ANYONE;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.extension.trace.propagation.OtTracePropagator;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.testing.exporter.InMemoryMetricReader;
+import io.opentelemetry.sdk.testing.exporter.InMemorySpanExporter;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import okhttp3.CertificatePinner;
+import okhttp3.ConnectionPool;
+import okhttp3.OkHttpClient;
 
 public class Main {
```

Comment:
I admit this file is a mess. It's overdue to split this up into multiple demo cases, I can take care of this.